tests/upgrade/test_crossgrade.sh

```diff
@@ -33,6 +33,7 @@ usage() {
     echo "  auth_enable   enable mtls."
     echo "  skip_cleanup  leave install intact after test completes."
     echo "  namespace     namespace to install istio control plane in (default istio-system)."
+    echo "  cloud         cloud provider name"
```

Comment:
Add allowed values for cloud and (required).
Also update example below.

Reply:
done!